src/main/java/org/example/daos/ApplicationDao.java

```diff
@@ -0,0 +1,72 @@
+package org.example.daos;
+
+import org.example.models.ApplicationRequest;
+import org.example.models.ApplicationResponse;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class ApplicationDao {
+
+    private final int s3LinkStatementIndex = 3; // This is needed for linter
+
+    public int createApplication(final ApplicationRequest applicationRequest)
+        throws SQLException {
+        try (Connection connection = DatabaseConnector.getConnection()) {
+            int linkParameterIndex = 2 + 1; // This is required for linter
```

Comment:
Is there not a cleaner way to write this?

Reply:
This has been corrected and committed.